src/Autosuggest.js

```diff
@@ -156,6 +159,14 @@ class Autosuggest extends Component {
     }
   }
 
+  maybeSelectFirstSuggestion() {
+    const { selectFirstSuggestion, multiSection, updateFocusedSuggestion } = this.props;
+
+    if (selectFirstSuggestion) {
+      updateFocusedSuggestion(multiSection ? 0 : null, 0);
+    }
+  }
```

Comment:
I agree that would be an improvement, but it's probably better to change it post-merge. There's a lot invested in this PR now and it would be wise not to introduce more bugs and testing so late in the game.